modules/candidate_list/jsx/openProfileForm.js

```diff
@@ -135,7 +149,6 @@ class OpenProfileForm extends Component {
         <ButtonElement
           name='Open Profile'
           label='Open Profile'
-          onUserInput={this.validateAndSubmit}
```

Comment:
This fires a second validation process (the first is fired by the form onSubmit event).
 A fetch network error occurs on the second run.